Project1/MoveHelper.h

```diff
@@ -1,8 +1,12 @@
 #pragma once
+#include <vector>
 #include "Map.h"
-#include "CellPath.h"
-class MoveHelper
-{
+
+class Map;
+class CellPath;
+class GridCell;
+class Unit;
+class MoveHelper {
```

Comment:
I am refactoring this in a new branch.

It isn't a subclass because i want to have different helper interfaces for movement, combat, and skills. and on map cannot branch into multiple inheritance directions.

I am making a MapHelper class that the other interfaces will inherit.